srcpkgs/system-config-printer/template

```diff
@@ -1,28 +1,28 @@
 # Template file for 'system-config-printer'
 pkgname=system-config-printer
-version=1.5.11
-revision=7
+version=1.5.13
+revision=1
 build_style=gnu-configure
+pycompile_dirs="usr/share/system-config-printer"
 configure_args="--with-udev-rules"
 make_build_args="udevhelperdir=/usr/lib/udev"
-make_install_args="udevhelperdir=/usr/lib/udev udevrulesdir=/usr/lib/udev/rules.d"
+make_install_args="udevhelperdir=/usr/lib/udev
+ udevrulesdir=/usr/lib/udev/rules.d"
```

Comment:
I don't think this line needs to be broken.

Reply:
It's 82 char long, but ok :)